app/forms/instance-create.tsx

```diff
@@ -138,6 +140,12 @@ export function CreateInstanceForm() {
   const image = allImages.find((i) => i.id === imageInput)
   const imageSize = image?.size ? Math.ceil(image.size / GiB) : undefined
 
+  useEffect(() => {
+    if (createInstance.error) {
+      setIsSubmitting(false)
+    }
+  }, [createInstance])
+
```

Comment:
Clean! Might be better to put `createInstance.error` directly in the deps instead of `createInstance`. I don’t know the behavior of RQ off the top of my head with respect to the top level object.

Reply:
Checked it out (turns out if you just use an existing instance name you can trigger the error state) and `createInstance.error` works well there.